src/views/partials/navbar.ejs

```diff
@@ -62,6 +62,12 @@
           <p>Emergency Contacts</p>
         </a>
       </li>
+      <li class="nav-item">
```

Comment:
Can you add a check here so that we only show this navbar group if `locals.discordAuthURL` is defined?